docker-compose.yml

```
@@ -0,0 +1,30 @@
+version: '2.1'
+services:
+  trup:
+    image: trup
+    build: ./
+    container_name: trup
+    env_file:
+      - .env
+    environment:
+      DATABASE_URL: "postgresql:///trup_postgres/?host=trup_postgres&database=trup"
```

```suggestion
      DATABASE_URL: "postgresql://trup_postgres/trup"
```
couldn't this also work? the reason the url in nix-shell is so weird is because of some extra shenanigans that have to do with the database running on a unix socket

---

I'll try